### ISSUE TITLE
[Coveralls] Increase the coveralls percentage

### ISSUE THREAD

**Issue by eswarasai** [MARKETProtocol/dApp#167]
## Before you `start work`

Please read our contribution [guidelines](https://docs.marketprotocol.io/#contributing) and if there is a bounty involved  please also see [here](https://docs.marketprotocol.io/#gitcoin-and-bounties)

If you have ongoing work from other bounties with us where funding has not been released, please do not pick up a new issue.  We would like to involve as many contributors as possible and parallelize the work flow as much as possible.

Please make sure to comment in the issue here immediately after starting work so we know your plans for implementation and a timeline.

Please also note that in order for work to be accepted, all code must be accompanied by test cases as well.

### Description

Currently our test coverage is at ~77%. We would like to bring test coverage as close to 100% as possible and start enforcing that PRs do not lower test coverage.

*Type*: Feature

### Definition of Done
- [ ] Raise coverage to >95% (or as close as possible)

**Comment by arun1595** [comment]
@eswarasai @pelsasser I would like to work on this!

**Comment by eswarasai** [comment]
@arun1595 -- There are few other things in the pipeline before we can scope this out. Thanks for your help. Will keep you posted once it's status is available.

**Comment by arun1595** [comment]
@eswarasai Ah, I see. Thanks for letting me know,

**Comment by pelsasser** [comment]
@eswarasai - can we open this back up a bit since some of the functionality has been merged?  if not yet, no problem.  Just trying to make sure we utilize all the hands that want to help out!

**Comment by pelsasser** [comment]
@eswarasai - can we update this issue / unblock and perhaps get a bounty staked for this issue?

**Comment by travisdmathis** [comment]
Closing this issue as we have begun breaking it down into multiple issues.
